From 9844abffef2e5dde6db4b121c988c2fb8ec82456 Mon Sep 17 00:00:00 2001
From: botantony <antonsm21@gmail.com>
Date: Wed, 22 Jul 2026 02:50:28 +0200
Subject: [PATCH] Add missing `p11_ver.h` file to installed headers

Running `make install` does not install `p11_ver.h` header which is used
by `p11_err.h`. This PR fixes it

Found in https://github.com/Homebrew/homebrew-core/pull/294427

Signed-off-by: botantony <antonsm21@gmail.com>
---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 92d3fdb..549ac67 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -9,7 +9,7 @@ EXTRA_DIST = Makefile.mak libp11.rc.in pkcs11.rc.in
 
 # Headers
 noinst_HEADERS= libp11-int.h pkcs11.h p11_pthread.h provider_helpers.h util.h
-include_HEADERS= libp11.h p11_err.h
+include_HEADERS= libp11.h p11_err.h p11_ver.h
 
 lib_LTLIBRARIES = libp11.la
 enginesexec_LTLIBRARIES =
-- 
2.55.0

